crates/re_space_view_time_series/src/view_part_system.rs

```diff
@@ -140,46 +130,40 @@ impl TimeSeriesSystem {
                 TimeRange::new(i64::MIN.into(), i64::MAX.into()),
             );
 
-            let components = Self::archetype_array();
-            let ent_views = range_entity_with_primary::<re_components::Scalar, 6>(
-                store, &query, ent_path, components,
-            );
+            let arch_views = range_archetype::<
+                TimeSeriesScalar,
+                { TimeSeriesScalar::NUM_COMPONENTS },
+            >(store, &query, ent_path);
```

Comment:
weird formatting O.o

Reply:
trying, but can't get it to format differently without introducing a type alias